chia/full_node/weight_proof.py

```diff
@@ -1606,41 +1628,41 @@ def map_segments_by_sub_epoch(sub_epoch_segments) -> Dict[int, List[SubEpochChal
     return segments
 
 
-def validate_total_iters(
-    segment: SubEpochChallengeSegment,
-    sub_slot_data_idx,
-    expected_sub_slot_iters: uint64,
-    finished_sub_slots_since_prev: int,
-    prev_b: SubSlotData,
-    prev_sub_slot_data_iters,
-    genesis,
-) -> bool:
-    sub_slot_data = segment.sub_slots[sub_slot_data_idx]
-    if genesis:
-        total_iters: uint128 = uint128(expected_sub_slot_iters * finished_sub_slots_since_prev)
-    elif segment.sub_slots[sub_slot_data_idx - 1].is_end_of_slot():
-        assert prev_b.total_iters
-        assert prev_b.cc_ip_vdf_info
-        total_iters = prev_b.total_iters
-        # Add the rest of the slot of prev_b
-        total_iters = uint128(total_iters + prev_sub_slot_data_iters - prev_b.cc_ip_vdf_info.number_of_iterations)
-        # Add other empty slots
-        total_iters = uint128(total_iters + (expected_sub_slot_iters * (finished_sub_slots_since_prev - 1)))
-    else:
-        # Slot iters is guaranteed to be the same for header_block and prev_b
-        # This takes the beginning of the slot, and adds ip_iters
-        assert prev_b.cc_ip_vdf_info
-        assert prev_b.total_iters
-        total_iters = uint128(prev_b.total_iters - prev_b.cc_ip_vdf_info.number_of_iterations)
-    total_iters = uint128(total_iters + sub_slot_data.cc_ip_vdf_info.number_of_iterations)
-    return total_iters == sub_slot_data.total_iters
+# def validate_total_iters(
+#    segment: SubEpochChallengeSegment,
+#    sub_slot_data_idx,
+#    expected_sub_slot_iters: uint64,
+#    finished_sub_slots_since_prev: int,
+#    prev_b: SubSlotData,
+#    prev_sub_slot_data_iters,
+#    genesis,
+# ) -> bool:
+#    sub_slot_data = segment.sub_slots[sub_slot_data_idx]
+#    if genesis:
+#        total_iters: uint128 = uint128(expected_sub_slot_iters * finished_sub_slots_since_prev)
+#    elif segment.sub_slots[sub_slot_data_idx - 1].is_end_of_slot():
+#        assert prev_b.total_iters
+#        assert prev_b.cc_ip_vdf_info
+#        total_iters = prev_b.total_iters
+#        # Add the rest of the slot of prev_b
+#        total_iters = uint128(total_iters + prev_sub_slot_data_iters - prev_b.cc_ip_vdf_info.number_of_iterations)
+#        # Add other empty slots
+#        total_iters = uint128(total_iters + (expected_sub_slot_iters * (finished_sub_slots_since_prev - 1)))
+#    else:
+#        # Slot iters is guaranteed to be the same for header_block and prev_b
+#        # This takes the beginning of the slot, and adds ip_iters
+#        assert prev_b.cc_ip_vdf_info
+#        assert prev_b.total_iters
+#        total_iters = uint128(prev_b.total_iters - prev_b.cc_ip_vdf_info.number_of_iterations)
+#    total_iters = uint128(total_iters + sub_slot_data.cc_ip_vdf_info.number_of_iterations)
+#    return total_iters == sub_slot_data.total_iters
```

Comment:
should I delete these functions instead of commenting them out?

Reply:
you can remove them

Reply:
done